analyzer/emerald/incoming.go

```diff
@@ -347,19 +347,18 @@ func extractRound(sigContext signature.Context, b *block.Block, txrs []*sdkClien
 	return &blockData, nil
 }
 
-func emitRoundBatch(batch *pgx.Batch, chainAlias string, round int64, blockData *BlockData) {
+func emitRoundBatch(batch *pgx.Batch, round int64, blockData *BlockData) {
 	for _, transactionData := range blockData.TransactionData {
 		for _, signerData := range transactionData.SignerData {
-			batch.Queue("INSERT INTO transaction_signer (chain_alias, height, tx_index, signer_index, addr, nonce) VALUES ($1, $2, $3, $4, $5, $6)", chainAlias, round, transactionData.Index, signerData.Index, signerData.Address, signerData.Nonce)
+			batch.Queue("INSERT INTO oasis_3.emerald_transaction_signers (round, tx_index, signer_index, signer_address, nonce) VALUES ($1, $2, $3, $4, $5)", round, transactionData.Index, signerData.Index, signerData.Address, signerData.Nonce)
```

Comment:
ya, in #158 we'll change this to use the same kind of query factory that the rest of the codebase uses.

rip sql ide features tho :cry:

Reply:
(Not a serious proposal; more work than it's worth; but might be cheaper to do for the emerald codebase)

We _could_ refactor the [various SQL-emitting functions](https://github.com/oasisprotocol/oasis-indexer/blob/215be5644aa3b63ba6075146fa45eeb643f8bf6a/analyzer/queries.go#L381) so that they are simple const strings that serve as templates. :smiling_imp: 

Something like
```
const (
  paratimeDeposit = "INSERT INTO {{.ChainId}}.{{.ParatimeName}}_deposits (round, sender, receiver, amount, nonce) VALUES ($1, $2, $3, $4, $5)"
)
```

Then proxy all SQL calls thru a layer that substitutes `{{...}}` for the real value.

Readability would improve a little ...